plugins/jobs/src/js/pages/JobDetailPageContainer.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { componentFromStream, graphqlObservable } from "data-service";
+import { Observable, BehaviorSubject } from "rxjs";
+import gql from "graphql-tag";
```

Comment:
so this file is the mediator? why is it in `pages` folder? 😕

Reply:
Because it's a page container. As far as I have seen the page containers are always right beside the pages. Where would you have searched for it?

Reply:
repository mediators are in `js` folder (one up), Dimas and my own Mediator are there 🤔